ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigCapellaImpl.java

```diff
@@ -48,6 +48,11 @@ public Optional<SpecConfigCapella> toVersionCapella() {
     return Optional.of(this);
   }
 
+  @Override
+  public int getMaxWithdrawalsPerPayload() {
+    return 16;
+  }
```

Comment:
i'd say we want to get this at the right point from the SpecVersion? it should be defined i would have thought, the capella config is in and loaded...

Reply:
fixed in #6372 fyi